internal/executor/application.go

```diff
@@ -74,7 +74,8 @@ func StartUpWithContext(config configuration.ExecutorConfiguration, clusterConte
 		clusterContext,
 		queueClient,
 		config.Kubernetes.MinimumPodAge,
-		config.Kubernetes.FailedPodExpiry)
+		config.Kubernetes.FailedPodExpiry,
+		config.Kubernetes.MinimumJobSize)
```

Comment:
Maybe we should just pass in KubernetesConfiguration here?

Reply:
Not sure about this one, there are still few unrelated things in the Kubernetes config.